web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -51,7 +51,6 @@
 import org.cbioportal.model.Patient;
 import org.cbioportal.model.Sample;
 import org.cbioportal.model.SampleList;
-import org.cbioportal.model.StructuralVariant;
```

Comment:
The StructuralVariant mixin is removed from the CustomObjectMapper in the web package .. but it does not seem to be connected to any object mapper in weblegacy. (the mixin class is moved below). Should we have a separation of weblegacy controller mixins and web controller mixins?  (for example, why is CopyNumberSegmentMixin still connected here?)

We can go forward, but if we are relocating the mixins for legacy controllers, we should set that up soon.